### DISCUSSION TITLE
2022-09-23, Version 3.3.0 @fengmk2

### DISCUSSION THREAD

**Discussion by fengmk2** [eggjs/egg#5030] [Announcements]
### Notable Changes

* **features**
  * Support config `serverGracefulIgnoreCode` to ignore error avoid process exit when uncatch error emit
  See https://github.com/node-modules/graceful/pull/13
### Commits

  * [[`a0761d65f`](http://github.com/eggjs/egg/commit/a0761d65f5df1002853c169efedab969636247d3)] - feat(graceful): support serverGracefulIgnoreCode (#5027) (hyj1991 <<yeekwanvong@gmail.com>>)
  * [[`8b8dd3be9`](http://github.com/eggjs/egg/commit/8b8dd3be95bb53ad3c732b8bc9c20566021e955f)] - chore: remove jsdoc and disable vercel comment (#5026) (Suyi <<thonatos.yang@gmail.com>>)
  * [[`f4225339f`](http://github.com/eggjs/egg/commit/f4225339f6235f78fe53d34d1eb0993faa410b36)] - test: fix ci (#5025) (TZ | 天猪 <<atian25@qq.com>>)
  * [[`5de994b9c`](http://github.com/eggjs/egg/commit/5de994b9c4cd17f9ecd4d4083c20b29f399a9e40)] - chore: fix action for gh-pages (#5024) (Suyi <<thonatos.yang@gmail.com>>)

<hr /><em>This discussion was created from the release <a href='https://github.com/eggjs/egg/releases/tag/3.3.0'>2022-09-23, Version 3.3.0 @fengmk2</a>.</em>

**Comment by ugvgylkm** [comment]
1. egg3.0版本现在有文档么，没见呢 这里好像也没有最新文档https://www.yuque.com/egg
2. egg3.0以上版本支持服务器渲染vue react么，要中间加一层nuxt么
